octobot_trading/data/portfolio_profitability.py

```diff
@@ -336,3 +351,6 @@ async def _get_currency_value(self, portfolio, currency, currencies_values=None,
                 return await self._evaluate_value(currency, portfolio[currency][CONFIG_PORTFOLIO_TOTAL], raise_error)
         return 0
 
+    def _should_currency_be_considered(self, currency, portfolio, ignore_missing_currency_data):
+        return (currency not in self.missing_currency_data_in_exchange or ignore_missing_currency_data) and \
+               (portfolio[currency][PORTFOLIO_TOTAL] or currency in self.traded_currencies)
```

Comment:
Just for clarity, can we use `portfolio[currency][PORTFOLIO_TOTAL] > 0` instead of `portfolio[currency][PORTFOLIO_TOTAL]` please ?